client/extensions/woocommerce/woocommerce-services/api/url.js

```diff
@@ -8,5 +8,6 @@ export const labelRefund = ( orderId, labelId ) => `connect/label/${ orderId }/$
 export const labelsPrint = () => 'connect/label/print';
 export const labelsTestPrint = () => 'connect/label/preview';
 export const addressNormalization = () => 'connect/normalize-address';
+export const shippingClasses = () => 'products/shipping_classes';
```

Comment:
It looks like this is unused.

Reply:
Yes, this is a left-over from before I switched to the data-layer. Now I removed it.